### PR TITLE
KOGITO-7312: Patch PR for KOGITO-7311 with doc improvements in Mocking HTTP CloudEvents sink with Wiremock guide

### PR DESCRIPTION
**Pull request by hmanwani-rh**
**JIRA:**https://issues.redhat.com/browse/KOGITO-7312

**Description:**Patch PR for KOGITO-7311 with doc improvements in Mocking HTTP CloudEvents sink with Wiremock guide

https://github.com/kiegroup/kogito-docs/pull/55/

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

**Comment by hmanwani-rh** [comment]
@nmirasch Please review the changes in the PR and let me know if the changes look good.

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc [hunk 1]
```diff
@@ -264,29 +266,33 @@ Now, the test needs to check the sink is receiving the expected events with:
...
+
+.Example of verifications
 [source]
 ----
 sink.verify(1, postRequestedFor(urlEqualTo("/")).withRequestBody(containing("\"type\":\"fraudEvaluation\"").and(containing("\"id\":\"" + order.getId() + "\""))));
 sink.verify(1, postRequestedFor(urlEqualTo("/")).withRequestBody(containing("\"type\":\"domesticShipping\"").and(containing("\"id\":\"" + order.getId() + "\""))));
 ----
 
-After assert the validations on received events, the test ends with success and the used WireMockServer will stop.
+After declaring the verfications on received events, the test successfully ends and the WireMockServer stops.
```
Comment (nmirasch) [inline serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:288]: After declaring the verification ..

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc [hunk 2]
```diff
@@ -264,29 +266,33 @@ Now, the test needs to check the sink is receiving the expected events with:
                 });
 ----
 
-{awaitility_class_url}[`await()`] Allows the test to retry the validations until assert the verifications, or until the specified time expires (in this case after 60 seconds).
+The `await()` method in the previous example allows the test to retry the validations until the verifications are declared or until the specified time is expired. In this example, the specified time is 60 seconds. 
 
-Checking if the sink (WireMockServer) receives two events for that order id with:
+The following example shows how to check if the sink (`WireMockServer`) receives two events for the same order ID:
+
+.Example if sink recevies the events
```
Comment (nmirasch) [inline serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:273]: Example if sink receives..

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc [hunk 3]
```diff
@@ -151,84 +153,83 @@ public class VerifyWorkflowExecutionIT { <2>
...
+.Workflows in `serverless-workflow-order-processing` example
 image::testing-and-troubleshooting/order-example-worflows.png[]
 
-The main workflow process the incoming Order event and start a parallel state calling two subflows: Fraud Handling and Shipping Handling.
-The workflow will end once **both** subflows end.
+The *Order Workflow* in the `serverless-workflow-order-processing` example application processes the incoming order event and starts a parallel state, which sends requests to two workflows including *Fraud Handling* and *Shipping Handling*. The Order Workflow ends when both Fraud Handling and Shipping Handling workflows are completed.
 
-Fraud Handling will produce a new `FraudEvaluation` event if the order is above 1000 USD.
-Any other system or service in the architecture can then read this event and react upon it, like canceling the order for example.
+The Fraud Handling worklflow produces a `FraudEvaluation` event if the received order is more than 1000 USD. In the workflow architecture, any other system or service can read the `FraudEvaluation` event and react upon it, such as canceling the order.
```
Comment (nmirasch) [inline serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:214]: The Fraud Handling workflow..

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc [hunk 4]
```diff
@@ -28,71 +27,74 @@ Mocking HTTP CloudEvents Sink with WireMock
...
+The Serverless Workflow application that you want to test must be configured to use link:{knative_eventing_url}[Knative Eventing]. You must configure Knative Eventing using standard HTTP POST requests to send and receive events between event producers and link:{knative_sink_binding_overview_url}[sinks]. The events between the event producers and sinks follow the link:{cloud_events_url}[CloudEvents] specification, which enables creating, parsing, sending, and receiving events in any programming language.
 
-The purpose of this guide is to mock that service to verify if the Cloud Events are correctly received by the sink.
-The {wiremock_url}[*WireMok*] framework will add the mocked server that will verify the different Cloud Events received by the sink in the Serverless Workflow service execution.
---
+When you create an event source, you can specify a sink where events are sent to, from the source. A sink is an transferable or a callable resource that can receive incoming events from other resources. The examples of sink include Kubernetes deployments, Knative services, channels, and brokers.
```
Comment (nmirasch) [inline serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:39]: ... A sink is a transferable..

**Review by nmirasch — APPROVED** [review]
@hmanwani-rh I just added small typos but the content looks good to me. Thank you very much, great job!

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:288]: Done

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc (hunk 2 above)
Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:273]: Done

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc (hunk 3 above)
Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:214]: Done

File: serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc (hunk 4 above)
Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc:39]: Done

**Comment by hmanwani-rh** [comment]
> @hmanwani-rh I just added small typos but the content looks good to me. Thank you very much, great job!

@nmirasch Thanks for reviewing the changes and catching typos. I've fixed all the typos. 
Therefore, merging this PR.

**Pull request merged**